tests/test_as_functiontype.py

```
@@ -8,4 +8,4 @@
 def test_as_functiontype():
     with pytest.raises(TypeError):
         as_functiontype(all)
-    assert as_functiontype(test_as_functiontype) is test_as_functiontype  # type: ignore[comparison-overlap]
+    assert as_functiontype(test_as_functiontype) is test_as_functiontype
```

Does the stuff that come after a `with pytest.raises: ` block actually get executed? I've been assuming so in my own tests as well, but now I'm starting to wonder if that's actually the case 🤔

---

yes it does, the `with raises` suppresses the exception